plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java

```diff
@@ -1408,6 +1515,9 @@ public static void register() {
         // @returns ListTag(ChunkTag)
         // @description
         // Gets a list of all chunks partially or entirely within the CuboidTag.
+        // @example
+        // # Loads the chunks that are within the cuboid "my_cuboid", even if they are partially within the cuboid.
```

Comment:
The point of examples should be to, well, *give an example*! This example meta is just reiterating what the description is, without adding any real example data. Should have some numbers or something that makes clear what it means to be 'partially' within vs fully